creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -347,27 +380,37 @@ class SnapbackSM {
         newReplicaSetSPIds.slice(1) // [secondary1, secondary2]
       )
 
+      response.issuedReconfig = true
+
       // Sync data from existing primary to new secondary
       phase = issueUpdateReplicaSetOpPhases.ENQUEUE_SYNCS
+      await this.enqueueSync({
```

Comment:
looks like the relationship of the node to the currently observed user doesn't matter? why cant you call export on any node for a given user? let's chat via video on this + other comments here

Reply:
this flow is still weird bc we will be enququing a sync to ourself via snapback, we should at least document that this is happening. the only reason i think its ok to do it this way is bc we don't have an easier way currently to enqueue a sync to self, ideally we would be able to call that function directly instead of making a roundabout + slower web request, but fine for now

Reply:
discussed offline -- leave as is for merge. we should have a followup PR to handle this case and not require a web request to enqueue a sync.

Reply:
will add a todo comment